docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,305 @@
+    ....
+
+    alertmanager:
+      enabled: false
+    ....
+
+    vmalert:
+      enabled: false
+    .....
+
+    vmagent:
+      spec:
+        selectAllByDefault: true
+        image:
+          tag: v1.91.3
+        scrapeInterval: 25s
+        externalLabels:
+          cluster: <cluster-name>
+    ....
+
+    defaultDashboardsEnabled: true
+
+    .....
+
+    kube-state-metrics:
+      enabled: true
+      ## all values for kube-state-metrics helm chart can be specified here  
+      ## Customizaing kube-state-metrics installation for scraping Custom resources related to Percona Operators
+      metricLabelsAllowlist:
+      - pods=[app.kubernetes.io/component,app.kubernetes.io/instance,app.kubernetes.io/managed-by,app.kubernetes.io/name,app.kubernetes.io/part-of],persistentvolumeclaims=[app.kubernetes.io/component,app.kubernetes.io/instance,app.kubernetes.io/managed-by,app.kubernetes.io/name,app.kubernetes.io/part-of],jobs=[app.kubernetes.io/component,app.kubernetes.io/instance,app.kubernetes.io/managed-by,app.kubernetes.io/name,app.kubernetes.io/part-of]
+      extraArgs: 
+      - --custom-resource-state-config-file=/go/src/k8s.io/kube-state-metrics/config
+      volumeMounts: 
+      - mountPath: /go/src/k8s.io/kube-state-metrics/
+        name: cr-config
+      volumes: 
+      - configMap:
+          name: customresource-config-ksm
+        name: cr-config  
+      rbac: 
+        extraRules:
+        - apiGroups:
+          - pxc.percona.com
+          resources:
+          - perconaxtradbclusters
+          - perconaxtradbclusters/status
+          - perconaxtradbclusterbackups
+          - perconaxtradbclusterbackups/status
+          - perconaxtradbclusterrestores
+          - perconaxtradbclusterrestores/status
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - psmdb.percona.com
+          resources:
+          - perconaservermongodbs
+          - perconaservermongodbs/status
+          - perconaservermongodbbackups
+          - perconaservermongodbbackups/status
+          - perconaservermongodbrestores
+          - perconaservermongodbrestores/status
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - pg.percona.com
+          resources:
+          - perconapgbackups/status
+          - perconapgclusters/status
+          - perconapgrestores/status
+          - perconapgclusters
+          - perconapgrestores
+          - perconapgbackups
+          - pgclusters
+          - pgpolicies
+          - pgreplicas
+          - pgtasks
+          verbs:
+          - get
+          - list
+          - watch 
+
+    .....
+
+    coreDns:
+      enabled: false
+    ```
+
+6. Install the Victoria Metrics Operator. The `vm-k8s` value in the following command is the Operator name. Replace the `<namespace>` placeholder with your value. The Namespace must be the same as the Namespace for the Secret and ConfigMap.
```

Comment:
```
The `vm-k8s` value in the following command is the Operator name
```
Kindly change it to 
```
The `vm-k8s` value in the following command is the Release Name
```
vm-k8s can be any name.

Reply:
Updated